library/cwm/src/lib/cwm/widget.rb

```diff
@@ -258,6 +258,60 @@ def validate_wrapper(_widget_id, _event)
     end
   end
 
+  # Wrapper when combination of old hash based CWM definition needed to be used
+  # with new object based one. Useful e.g. when content is provided by other
+  # module like CWMFirewallInterfaces.CreateOpenFirewallWidget
+  # @note it does not support common methods and helpers from abstract widget
```

Comment:
Well, do you have other idea? Another option is to ignore call of that method, but I find it too error prone.